skepticoin/networking/peer.py

```diff
@@ -465,7 +470,11 @@ def check_message_backlog(self) -> None:
             self.start_sending()
 
     def start_sending(self) -> None:
-        self.local_peer.selector.modify(self.sock, selectors.EVENT_READ | selectors.EVENT_WRITE, data=self)
+        try:
+            self.local_peer.selector.modify(self.sock, selectors.EVENT_READ | selectors.EVENT_WRITE, data=self)
+        except ValueError:
+            # TODO Invalid file descriptor: -1 happens here
```

Comment:
Can we log the IP address of the invalid FD? That would help troubleshoot and investigate the reason for the -1.

Reply:
Done.